src/StackHCI/Az.StackHCI.psm1

```diff
@@ -4689,16 +4771,26 @@ function New-Directory{
 #>
 function Invoke-DeploymentModuleDownload{
     # Remote Support
-    New-Variable -Name RemoteSupportPackageUri -Value "https://remotesupportpackages.blob.core.windows.net/packages" -Option Constant -Scope Script
+    New-Variable -Name RemoteSupportPackageUri -Value "https://srsqldatasa.blob.core.windows.net/packages" -Option Constant -Scope Script
```

Comment:
Do not bring this change over (or anything related to RS except for logging/etc).